DNA-RNA.py

```diff
@@ -0,0 +1,58 @@
+seq = str(input()).upper()
+
+
+class RNA:
+    def __init__(self, seq):
+        self.seq = seq
+        self.length = len(seq)
+
+    def gc(self):
+        self.gc_count = 0
+        for i in range(self.length):
+            if (self.seq[i] == 'G') or (self.seq[i] == 'C'):
+                self.gc_count += 1
+        self.gc_content = str(int((self.gc_count / self.length) * 100)) + str('%')
```

Comment:
Why do you use "int" type? Can GC content be not integer?

Reply:
There just were too many digits after the decimal point=) I can round it up to two digits